modules/spring-extras/src/main/java/org/mule/module/spring/events/MuleEventMulticaster.java

```diff
@@ -412,6 +413,17 @@ else if (!(listener instanceof MuleEventListener))
         }
     }
 
+    /***
+     * Implemented only for compliance with Spring 4.2+
+     * Please use the {@link #multicastEvent(ApplicationEvent) multicastEvent} method.
+     * @param event
+     * @param eventType
+     */
+    @Override
+    public void multicastEvent(ApplicationEvent event, ResolvableType eventType) {
```

Comment:
is this related to the spring upgrade?

Reply:
Yes, that's why I mention in the Javadoc that we must implement this for compliance with the Spring 4.2+ ApplicationEventMulticaster interface that this class implements.